client/src/pages/SignIn.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { User } from '../../../types/User';
+
+export default function SignIn(): JSX.Element {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  async function loginUser(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    const user = {
+      email: email,
+      password: password,
+    };
+
+    const responseUser = fetch('/api/users/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(user),
+    });
```

Comment:
Shouldn't we show an error message if this request fails?

Reply:
Yes I am working on it